abd-clam/src/core/graph/criteria.rs

```diff
@@ -1,4 +1,4 @@
 //! Criteria used for selecting `Cluster`s for `Graph`s.
 
 /// A `Box`ed function that assigns a score for a given `Cluster`.
-pub type MetaMLScorer = fn(crate::cluster::Ratios) -> f64;
+pub type MetaMLScorer = Box<fn(crate::core::cluster::Ratios) -> f64>;
```

Comment:
Why box the function?

Reply:
The following block of code boxes the score functions and stores them in a vec but the type of MetaMLScorer is not boxed. I could change the type of the vec to be `Vec<(&'a str, Box<crate::core::graph::MetaMLScorer>)>` instead if you prefer

```
pub fn get_meta_ml_scorers<'a>() -> Vec<(&'a str, crate::core::graph::MetaMLScorer)> {
    vec![
        ("lr_manhattan_sc", Box::new(lr_manhattan_sc)),
```

Reply:
It should be doable without using box but I don't have a preference either way. I was just curious.

Reply:
Without boxing I get an error. The existing comment above the type alias refers to it as a boxed function as well. If you don't have a preference I'll just leave it as is in the commit